web/src/app/metalakes/metalake/rightContent/tabsContent/TabsContent.js

```diff
@@ -140,7 +140,7 @@ const TabsContent = () => {
                   .map((item, index) => {
                     return (
                       <CustomTooltip
-                        key={item.type}
+                        key={index}
```

Comment:
Why change the key to index?

Reply:
We are not going to sort this list, so the key being the index or item type does not matter.
I reverted it.